doc/user-guide/06-layout-configuration.adoc

```diff
@@ -143,61 +106,60 @@ we mount the +/dev/mapper/backup-backup+ filesystem on
 +/media/backup+, it will get excluded, as the mountpoint is under
 +/media+.
 
-=== Manual excludes
-It seems prudent to prevent the external drives from ever being backed-up or overwritten. The default configuration contains these lines:
+If we mount the filesystem on +/dev/mapper/backup-backup+ on +/backup+,
+Relax-and-Recover will think that it's necessary to recreate the filesystem:
 ----------------------------------
-# Exclude components from being backed up, recreation information is active
-EXCLUDE_BACKUP=()
-
-# Exclude components during component recreation
-# will be added to EXCLUDE_BACKUP (it is not backed up)
-# recreation information gathered, but commented out
-EXCLUDE_RECREATE=()
-
-# Exclude components during the backup restore phase
-# only used to exclude files from the restore.
-EXCLUDE_RESTORE=()
+disk /dev/sdc 1999696297984 msdos
+...
+part /dev/sdc 1073741824000 1048576 primary boot /dev/sdc1
+part /dev/sdc 925953425408 1073742872576 primary lvm /dev/sdc2
+lvmdev /dev/backup /dev/sdc2 cJp4Mt-Vkgv-hVlr-wTMb-0qeA-FX7j-3C60p5 1808502784
+...
+lvmgrp /dev/backup 4096 220764 904249344
+...
+lvmvol /dev/backup backup 12800 104857600 
+lvmvol /dev/backup externaltemp 38400 314572800 
+...
+fs /dev/mapper/backup-backup /backup ext4 uuid=da20354a-dc4c-4bef-817c-1c92894bb002 label= blocksize=4096 reserved_blocks=655360 max_mounts=24 check_interval=180d options=rw
 ----------------------------------
 
-To prevent an inadvertently mounted backup filesystem being added to the restore list, the easiest way is to add the filesystem to the +EXCLUDE_RECREATE+ array.
+=== Manual excludes
+It seems prudent to prevent the external drives from ever being backed-up or overwritten.
+The most generic mechanism for doing this is the +EXCLUDE_RECREATE+
+variable. It can contain a list of storage components that will be
+excluded from the layout (will be present in the layout file, but
+commented out) recursively together with their descendands (components
+that depend on the excluded component, like a filesystem on the
+underlying logical volume). Each component in the list corresponds to
+a component in the layout file, but the syntax is different. To
+specify a mounted filesystem, use the +fs:+ prefix:
++fs:+<mountpoint>. To specify swap, use the +swap:+ prefix. The syntax
+to specify a volume group to exclude (together with all its logical
+volumes and filesystems on them) is +/dev/+<volume group name>. Consult
+the +/var/lib/rear/layout/disktodo.conf+ file created together with
+the layout file for the full list of components in a compatible
+syntax.
+
+To prevent an inadvertently mounted backup filesystem being added to
+the layout file and recreated, the easiest way is to add the filesystem to the +EXCLUDE_RECREATE+ array.
```

Comment:
Ouch! Argh! Ugh! ;-)

Please remove all
* just
* simple
* easy

words from all documentation everywhere.

Those words are never helpful for a user.
Those words make a desperate user who reads documentation angry
because those words humiliate the user because they basically
tell him he is somewhat stupid when he has issues with things
that are just simple and easy for others.

It is in particular humiliating and patronizing
when the developers tell their users
that things are just simple and easy.

Users who read documentation are often desperate
because reading documentation is often the last thing
one does when (easy) things (simply) do not (just) work.

Reply:
@jsmeix I actually share your opinion on the general tone and style of this whole document (applies to most of your review comments), but it was not my goal to rewrite it entirely, only to improve its factual correctness and logical structure.

If you review the commits one by one, as suggested in https://github.com/rear/rear/pull/3125#issuecomment-1883420901 , you will see that most of the wording that you don't like has been there since the beginning.

Reply:
I know well that most of the wording I don't like
is there since the beginning.

So no need to worry now about the general tone and style.

Perhaps later I may (provided I find time for it)
clean up the general tone and style.

Reply:
Partly addressed in d1440889db7d162e47bc6cfa37e23709150c5456 (for the parts that I have touched).

Reply:
For the future I filed
https://github.com/rear/rear/issues/3129